### PR TITLE
adding swagger documentation for featured positions

### PR DESCRIPTION
**Pull request by SidelineCory24**
testing the parameters to verify that the correct data is returned

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Would anymore fields need to be added? The fields are the same as available positions/tandem.

**Comment by scott062** [comment]
This is a great start. Rather than making the swagger docs everything the python function could receive, let's limit it to only the params the frontend would pass for this functionality. This way a developer would never unintentionally pass nonsensical params for featured positions, even if the position search would "work." 

To figure out what the possible featured position params can be, you'll want to look in the FE actions for the url.

**Comment by SidelineCory24** [comment]
@scott062 changes made based off of what I found on the FE 🙏

File: talentmap_api/fsbid/views/available_positions.py [hunk 1]
```diff
@@ -188,6 +188,19 @@ def get(self, request, pk):
 class FSBidAvailablePositionsFeaturedPositionsView(BaseView):
 
     permission_classes = (IsAuthenticatedOrReadOnly,)
+    filter_class = AvailablePositionsFilter
+    schema = AutoSchema(
+        manual_fields=[
+            # Pagination
+            coreapi.Field("ordering", location='query', description='Ordering'),
+            coreapi.Field("page", location='query', description='Page Index'),
+            coreapi.Field("limit", location='query', description='Page Limit'),
```
Comment (scott062) [inline talentmap_api/fsbid/views/available_positions.py:197]: I believe these are hard coded in the function, so you can leave them off.

File: talentmap_api/fsbid/views/available_positions.py [hunk 2]
```diff
@@ -188,6 +188,19 @@ def get(self, request, pk):
...
+
+            coreapi.Field("id", location="query", description="Available Position ids"),
```
Comment (scott062) [inline talentmap_api/fsbid/views/available_positions.py:199]: Can remove id

File: talentmap_api/fsbid/views/available_positions.py [hunk 3]
```diff
@@ -188,6 +188,19 @@ def get(self, request, pk):
...
+            coreapi.Field("position__grade__code__in", location='query', description='Grade Code'),
+            coreapi.Field("position__skill__code__in", location='query', description='Skill Code'),
```
Comment (scott062) [inline talentmap_api/fsbid/views/available_positions.py:201]: These are the ones you pulled from the FE, there should be one more.

File: talentmap_api/fsbid/views/available_positions.py (hunk 1 above)
Comment (scott062) [reply talentmap_api/fsbid/views/available_positions.py:197]: Sorry, it looks like we pass limit so keep that one

**Comment by SidelineCory24** [comment]
this should do it😅

**Comment by scott062** [comment]
👍  Looks good. Just click the update branch button to bring it in sync with the most recent dev. Once it passes the circleci checks, then you will have the option to merge.

**Review by scott062 — APPROVED** [review]

**Pull request merged**